client_dashboard/common/src/commonMain/kotlin/org/thechance/common/domain/usecase/ManageTaxisUseCase.kt

```diff
@@ -15,7 +15,7 @@ interface IManageTaxisUseCase {
     suspend fun createTaxiReport()
 
     suspend fun getTaxis(
-        username: String? = null,
+        username: String?,
```

Comment:
why username is nullable if it's getting all taxis when it's nullable and by username when it's not null this fucntion should be seprated into two functions